content/src/main/content/jcr_root/etc/clientlibs/acs-commons/rte-rtl-switch/.content.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:ClientLibraryFolder"
+    categories="[rte.coralui2]"/>
```

Comment:
changed, any other steps that need to be performed so this can be included?
I think we should ensure that this only is included in granite ui, I did not test for side effects in classic ui.